usaspending_api/transactions/delta_models/transaction_search.py

```diff
@@ -101,9 +101,110 @@
     "funding_office_name": "STRING",
 }
 
+TRANSACTION_SEARCH_POSTGRES_COLUMNS = {
```

Comment:
I wonder if there's a way to not have to redefine ALL the columns again if most of them are not changing, but only transform one set of columns into the other (and still allow it to be referred to as a dict constant like this).

It also helps answer the question that anyone would ask when they see these two dicts: _What's the different between these two?_

Reply:
Yeah I agree this was thrown together hastily. I didn't want to get involved with updating the other scripts/tests that used those columns. I could standardize it if the backend devs didn't mind me updating it everywhere.

@sethstoudenmier @collinwr @emily-brents @bkschmidt